frontend/src/main/pages/HomePage.js

```diff
@@ -25,6 +26,20 @@ export default function HomePage() {
         url: "/api/commons/all"
       }
     );
+  const onSuccess = (commons) => {
+    // Stryker disable next-line all : hard to get variable
+    let existed = new Boolean(false);
+    for(let i = 0; i < commonsJoined.length; i++ ){
+      if(commonsJoined[i].id === commons.id){
+        existed = true;
+      }
+    }
+    if(existed === true){
```

Comment:
Comparing booleans is redundant. You can just do `if (existed)` here.

Reply:
changed